.gitignore

```diff
@@ -0,0 +1,2 @@
+workflows
```

Comment:
- [ ] Kindly replace workflows with node_modules as per the requirements of the project.